crates/napi/src/next_api/project.rs

```diff
@@ -1441,12 +1441,17 @@ pub struct OptionStackFrame(Option<StackFrame>);
 #[turbo_tasks::function]
 pub async fn get_source_map_rope(
     container: Vc<ProjectContainer>,
-    file_path: RcStr,
+    source_url: RcStr,
 ) -> Result<Vc<OptionStringifiedSourceMap>> {
-    let (file, module) = match Url::parse(&file_path) {
+    let (file_path_sys, module) = match Url::parse(&source_url) {
         Ok(url) => match url.scheme() {
             "file" => {
-                let path = urlencoding::decode(url.path())?.to_string();
+                let path = match url.to_file_path() {
+                    Ok(path) => path.to_string_lossy().into(),
+                    Err(error) => {
+                        bail!("Failed to convert file URL to file path: {error:?}");
+                    }
```

Comment:
Error is of type `()`, i.e. there are never any more details here. You want to instead print the `url`
```suggestion
                    Err(_) => {
                        bail!("Failed to convert file URL to file path: {url}");
                    }
```